specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/examples/ApiManagementGetApiSchema.json

```diff
@@ -0,0 +1,165 @@
+                                    },
+                                    "promo_code": {
+                                        "type": "string",
+                                        "description": "Promo code of the Uber user."
+                                    }
+                                }
+                            },
+                            "Activity": {
+                                "properties": {
+                                    "uuid": {
+                                        "type": "string",
+                                        "description": "Unique identifier for the activity"
+                                    }
+                                }
+                            },
+                            "Activities": {
+                                "properties": {
+                                    "offset": {
+                                        "type": "integer",
+                                        "format": "int32",
+                                        "description": "Position in pagination."
+                                    },
+                                    "limit": {
+                                        "type": "integer",
+                                        "format": "int32",
+                                        "description": "Number of items to retrieve (100 max)."
+                                    },
+                                    "count": {
+                                        "type": "integer",
+                                        "format": "int32",
+                                        "description": "Total number of items available."
+                                    },
+                                    "history": {
+                                        "type": "array",
+                                        "items": {
+                                            "$ref": "#/definitions/Activity"
+                                        }
+                                    }
+                                }
+                            },
+                            "Error": {
+                                "properties": {
+                                    "code": {
+                                        "type": "integer",
+                                        "format": "int32"
+                                    },
+                                    "message": {
+                                        "type": "string"
+                                    },
+                                    "fields": {
+                                        "type": "string"
+                                    }
+                                }
+                            },
+                            "ProductArray": {
+                                "type": "array",
+                                "items": {
+                                    "$ref": "#/definitions/Product"
```

Comment:
This reference cannot be resolved, see [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/283525487).

Reply:
That is just a Schema. It will differ from Api to API. I will update the example with a WSDL Api Schema, so that the tooling doesn't get confused.